src/ucp/wireup/select.c

```diff
@@ -673,9 +674,10 @@ ucp_wireup_tl_iface_latency(const ucp_worker_iface_t *wiface,
         local_lat = ucp_wireup_iface_lat_distance_v1(wiface);
         /* Address v1 contains just latency overhead */
         return ((local_lat + remote_iface_attr->lat_ovh) / 2) +
-               (wiface->attr.latency.m * context->config.est_num_eps);
+               (wiface->attr.latency.m *
+                ucp_context_get_est_num_eps(context, prioritized_ep));
```

Comment:
can we use here ucp_tl_iface_latency_with_priority?

Reply:
do you mean call `ucp_tl_iface_latency_with_priority` and substuct lat.c from the result?